src/UI/AddMovies/MinimumavailabilityTooltipTemplate.hbs

```diff
@@ -0,0 +1,8 @@
+<dl class="minimumavailability-tooltip-contents">
+    <dt>Announced</dt>
```

Comment:
preDB is not implemented and set as default.. for now it behaves identical to Physical/Web but its all ready for when preDB work is complete.... a couple comments with TODO: handle preDB differently here